REFERENCE.md

```diff
@@ -39,39 +39,53 @@ Data type: `Boolean`
 
 Enable or disable the fact deployment. Doesn't work on FreeBSD yet
 
+Default value: `true`
+
 ##### `manage_service`
 
 Data type: `Boolean`
 
 Enable or disable the service management
 
+Default value: `true`
+
 ##### `manage_repo`
 
 Data type: `Boolean`
 
 Enable or disable the repository setup
 
+Default value: `true`
```

Comment:
Just reasoning logically here. As a user I would assume this is true everywhere, but it's not. That means it's not correct.

Therefor I'd suggest the following:
* If everything uses the same value, put it in `init.pp`
* If there's a common value, but some override it, put it in `data/common.yaml`
* OS specific overrides go the appropriate `data/*.yaml` files.

We can apply the same logic to `manage_jq`.